daemon/daemon_main.go

```diff
@@ -1104,14 +1104,23 @@ func runDaemon() {
 	go d.nodeMonitor.Run(path.Join(defaults.RuntimePath, defaults.EventsPipe), bpf.GetMapRoot())
 
 	bootstrapStats.k8sInit.Start()
-	<-d.initK8sSubsystem()
-	bootstrapStats.k8sInit.End(true)
+	// We need to set up etcd in parallel so we will init the k8s subsystem
+	// and wait for all caches to be synced before restoring any endpoints.
+	// This is required because CNP with CIDRs rely on kvstore connectivity
```

Comment:
Please rewrite this description. It is not clear to me. How do CNP and CIDRs rely on kvstore connectivity?